map/src/data/dataDriver.ts

```diff
@@ -2,34 +2,23 @@ import {
   Location as BaseLocation,
   MarkerInfo as BaseMarkerInfo,
   MarkerInfoWithId as BaseMarkerInfoWithId,
-  MARKER_COLLECTION_ID,
-  MARKERS_STORAGE_PATH,
 } from '@reach4help/model/lib/markers';
-import * as firebase from 'firebase/app';
-
-// eslint-disable-next-line import/no-duplicates
-import 'firebase/firestore';
-// eslint-disable-next-line import/no-duplicates
-import 'firebase/storage';
-// eslint-disable-next-line import/no-duplicates
-import 'firebase/analytics';
```

Comment:
Until we integrate Algolia analytics, we shouldn't remove this.

Reply:
This will not affect firebase analytics.    Firebase is no longer used by dataDriver.ts (previously named fireBase.ts)